RJ_simplex_search_method.py

```diff
@@ -99,10 +99,26 @@ def simplex_method(f, list_of_vertices):
 
 # Input for the vertices
 list_of_vertices = []
-print("Enter the three vertices:")
-for i in range(3):
-    vertex = list(map(float, input(f"Enter vertex {i + 1} (e.g., 1 2): ").split()))
-    list_of_vertices.append(vertex)
+var=int(input("Enter the no of vertices max(3):"))
+if var==2:
+    print("Enter the two variable  on that must be on same line(ex: (2.5,4) (2.5,6)")
```

Comment:
Do they need to be in same line?
for sure??

Reply:
Two points are always collinear. 
Change your prompt message if I'm right.
Just check it, I'm not totally sure.